lib/libvcc/vcc_symb.c

```diff
@@ -297,7 +306,9 @@ VCC_SymbolGet(struct vcc *tl, vcc_ns_t ns, vcc_kind_t kind,
 		if (tn1->tok != '.')
 			break;
 		tn1 = vcc_PeekTokenFrom(tl, tn1);
-		if (tn1->tok != ID)
+		if (tn1->tok == CSTR && sym->wildcard == NULL)
```

Comment:
Flexelint complains that `sym` is possibly NULL here.  Add an assert to make sure.